### PR TITLE
Update run_schrod.rst

### PR DESCRIPTION
**Pull request by bjartelund**
no route to host for c1-3, perhaps c1-1 would be suitable?

### CODE REVIEW

**Comment by bast** [comment]
Thanks! @eta000 do you agree?

**Comment by sth000** [comment]
I think C1-3 died and we moved all the Schrödinger stuff to C1-2

Steinar

Sendt fra min iPhone

13. mai 2019 kl. 09:29 skrev Radovan Bast <notifications@github.com<mailto:notifications@github.com>>:


Thanks! @eta000<https://github.com/eta000> do you agree?

File: applications/chemistry/Schrodinger/run_schrod.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ This page contains info about how to submit Schrodinger jobs on Stallo, based
 on two sets of examples - one for docking and one for molecular dynamics -
 provided to us by users.
 
-Pay enhanced attention to the `ssh -Y c1-3` in examples below; this represents the adviced behaviour on how to run jobs on Stallo for your benefit solely!
+Pay enhanced attention to the `ssh -Y c1-1` in examples below; this represents the adviced behaviour on how to run jobs on Stallo for your benefit solely!
```
Comment (bast) [inline applications/chemistry/Schrodinger/run_schrod.rst]:
```suggestion
Pay enhanced attention to the `ssh -Y c1-2` in examples below; this represents the adviced behaviour on how to run jobs on Stallo for your benefit solely!
```

File: applications/chemistry/Schrodinger/run_schrod.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ Do the following:
 
 .. code-block:: bash
 
-   $ ssh -Y c1-3
+   $ ssh -Y c1-1
```
Comment (bast) [inline applications/chemistry/Schrodinger/run_schrod.rst]:
```suggestion
   $ ssh -Y c1-2
```

File: applications/chemistry/Schrodinger/run_schrod.rst [hunk 3]
```diff
@@ -59,7 +59,7 @@ Do the following:
...
```
Comment (bast) [inline applications/chemistry/Schrodinger/run_schrod.rst]:
```suggestion
   $ ssh -Y c1-2
```

**Comment by bast** [comment]
Thanks again!

**Pull request merged**